apps/cli/src/adapters/node/codec.ts

```diff
@@ -0,0 +1,27 @@
+import { z } from "zod";
```

Comment:
@kin0992 I am thinking we can start using `zod4`, the new API (https://zod.dev/v4). It should be already included in this NPM package (the import is a little bit different), the API is almost the same but is more performant and there are many "quality of life" upgrades

Give it a try!